src/components/donations/CryptoTaxCalculator.tsx

```
@@ -0,0 +1,154 @@
+    const [result, setResult] = useState<TaxCalculationResult | null>(null);
+
+    useEffect(() => {
+        async function loadPrices() {
+            const fetchedPrices = await getCurrentCryptoPrices();
+            setPrices(fetchedPrices);
+        }
+        loadPrices();
+    }, []);
+
+    useEffect(() => {
+        if (donationAmount > 0 && costBasis > 0) {
```

ESLint

[error] Replace `········` with `····`

(prettier/prettier)